src/porepy/numerics/fem/rt0.py

```diff
@@ -12,18 +12,18 @@
 
 import porepy as pp
```

Comment:
At some point, we should replace the long imports with simply pp.foo for all methods and classes that are exposed at the top level. But not today.